course_info/static/course_info/js/app.js

```diff
@@ -18,8 +18,7 @@
                     //
                     // window.globals.append_resource_link_id function added by
                     // django_auth_lti/js/resource_link_id.js
-                    if (!(config.url.startsWith('uib/template/alert/') ||
-                                config.url.startsWith('uib/template/modal/'))) {
+                    if (!(config.url.startsWith('uib/template/'))) {
```

Comment:
I think there's an unnecessary set of parens in there, I'm guessing because it was a quick edit.  This can just be

```javascript
if (!config.url.startsWith('uib/template/')) {
```

no?